tcms/locale/sl_SI/LC_MESSAGES/django.po

```diff
@@ -188,7 +171,7 @@ msgstr "Prijavi težavo"
 
 #: tcms/settings/common.py:125
 msgid "Ask for help on StackOverflow"
-msgstr "Postavite vprašanje glede uporabe Kiwi TCMS na StackOverflow"
+msgstr "Postavite vprašanje glede uporabe KiwiTCMS na StackOverflow"
```

Comment:
@Prome88 why are you changing this to "KiwiTCMS" ? The official project name is "Kiwi TCMS" - two words so we'd like to keep it this way.

Reply:
Im not changing it. The string "KiwiTCMS" is on Crowdin from 7 months ago and was not changed since. I'll fix it ASAP.

Reply:
hmmm, I've updated this in 7e7ecbae but somehow Crowdin didn't get synchronized :(